.travis.yml

```diff
@@ -18,4 +18,4 @@ deploy:
   skip_existing: true
   on:
     tags: true
-    repo: amundsen-io/amundsendatabuilder
+    repo: lyft/amundsendatabuilder
```

Comment:
why change it back to lyft?

Reply:
This was in error, I believe because I rebased incorrectly after the other PR changing this landed. Fixed, there are now 0 references to lyft in the repo